### DISCUSSION TITLE
Policy templates for Firefox 89 and Firefox ESR 78.11

### DISCUSSION THREAD

**Discussion by mkaply** [mozilla/policy-templates#775] [General]
Note: While these templates will work for Firefox ESR68, they contain new policies that are not in Firefox ESR 68. If you need to manage Firefox ESR 68, you should use [v1.17](https://github.com/mozilla/policy-templates/releases/tag/v1.17)

The README that corresponds to this release is:

https://github.com/mozilla/policy-templates/blob/v2.11/README.md

This release provides support for the following new policies:

[AllowedDomainsForApps](https://github.com/mozilla/policy-templates/blob/master/README.md#alloweddomainsforapps)

This release updates the following policies:

[ExtensionSettings](https://github.com/mozilla/policy-templates/blob/master/README.md#extensionsettings) - Updated to allow disabling updates to individual extensions with the updates_disabled option.

[Preferences](https://github.com/mozilla/policy-templates/blob/master/README.md#preferences) - Added additional prefixes: gfx. and layers.

For more detailed information about Firefox 89 and Firefox 78.11 ESR, see the [regular release notes](https://www.mozilla.org/firefox/89.0/releasenotes/) and the [enterprise release notes](https://support.mozilla.org/kb/firefox-enterprise-89-release-notes).

<hr /><em>This discussion was created from the release <a href='https://github.com/mozilla/policy-templates/releases/tag/v2.11'>Policy templates for Firefox 89 and Firefox ESR 78.11</a>.</em>

**Comment by Droidiferus** [comment]
Hi, would it be to hard to use a fully defined  DATE  such as June 1, 2021,  instead of just  Jun 1  ?  
Because it leaves the user guessing whether the post is ancient or current.
Thank You!

PS: Yes, I know, for the superstars, things go without saying, but some people are not software superstars.  In fact, 99.999999999% of people on this planet are not software superstars, but just want to know what the words mean, that are coming out of your and your
peers collective mouths.   Why are common courtesy and explanations so hard to obtain from software people?

**Reply by mkaply** [reply]
You'll have to report that problem to Github, since it's their issue.

But in general, when you see a site that has a month/day with no year, it's the current year and you can hover over to see a popup with the date.
